monster-js/xiaoqingb/js/daguaishou.js

```diff
@@ -0,0 +1,115 @@
+// 初始血量
+var newblood=100;
+// 获取复活按钮，初始不激活
+var reviveEle=document.getElementById("revive");
+// 获取血量
+var bloodEle=document.getElementById("remain-blood");
+// 获取血条
+var volumnEle=document.getElementById("blood-volumn");
+// 定义出拳及复活次数
+var myGuess,computerGuess,num=3;
+// 伤害函数
+function hert(){
+    // 设置0为下限
+    if(newblood<=0)
+    {
+        bloodEle.innerHTML="dead";
+        document.getElementById("blood-volumn").style.width=0;
+        // 弹窗询问是否复活,前提是还有复活次数
+        if(num>0){
+            if (confirm("他已经死翘翘了，要复活他不？")){
+                alert("想救人？？？需进行猜拳");
+                document.getElementById("Rock-paper-scissors").style.visibility="visible";
+                // var ui = document.getElementById("Rock-paper-scissors");
+                // ui.style.visibility="visible";
+                // reviveEle.onclick=bloodMaxed();
+                // reviveEle.style.background="#FF6666";
+            }
+        }
+    }
+    else{
+        // 计算掉血后剩余血量
+        newblood-=bloodLoss();
+        document.getElementById("Rock-paper-scissors").style.visibility="hidden";
+        // 必须在这里提前置零，因为如果在弹窗处置零的话，会在弹窗后才会血量置零
+        if(newblood<0){
+            newblood=0;
+        }           
+        bloodEle.innerHTML="Hp:"+newblood;
+        document.getElementById("blood-volumn").style.width=newblood+"%";
+        
+    }
+}
+// 扣血量函数
+function bloodLoss(){
+    var a;
+    a=parseInt(Math.random()*(20-10)+10);
+    return a;
+}
+// 复活函数
+function bloodMaxed(){
+    newblood=100;
+    document.getElementById("blood-volumn").style.width=newblood+"%";
+    bloodEle.innerHTML=newblood;
+    console.log(newblood);
+    document.getElementById("dec").innerHTML="剩余复活次数："+(num-=1);
+    document.getElementById("Rock-paper-scissors").style.visibility="hidden";
+    if(num===0){
+        alert("复活次数已经用完，再打死就复活不了喽");
+    }
+}
+
+// 猜拳游戏 剪刀石头布分别为1，2，3
+// 电脑出拳
+function computer(){
+    computerGuess=parseInt(Math.random()*(4-1)+1);
```

Comment:
```suggestion
    return parseInt(Math.random()*(4-1)+1);
```
建议把这个电脑出拳的结果作为返回值返回，不要直接放到变量里面哈~